src/website/app/pages/Styling/styling.md

```diff
@@ -57,14 +66,18 @@ This function is used to create a new styled component based on another componen
 **Parameters**
 
  * `element`: A React component or a DOM element tag name
- * `styles`: An [object of style rules](https://github.com/threepointone/glamor/blob/master/docs/howto.md)
- or a function that accepts props and context and returns an object of style
- rules
- * `options`: Optional. An object containing a mix of Mineral UI and
- [Glamorous options](https://glamorous.rocks/api).  Common uses include
-    * setting a display name on your component: `{ displayName: 'MyComponent' }`,
-    * declaring which props to forward to the DOM element: `{ forwardProps: [href, customProp] }`,
-    * and including a style reset: `{ includeStyleReset: true }`
+ * `styles`: A [style rule object](https://emotion.sh/docs/css), an array of style rule objects, or a function that accepts props and context and returns either a style
+ rule object or an array of style rule objects.
+ * `options`: Optional. An object with the following shape. All properties are optional.
+
+| Option              | Type          | Description                                                                                        |
+|---------------------|---------------|----------------------------------------------------------------------------------------------------|
+| `displayName`       | String        | Sets component displayName and CSS className label during development                              |
+| `filterProps`       | Array<String> | Props that should not be passed to child components                                                |
+| `forwardProps`      | Array<String> | Props that should be passed to child components. Takes precedence over `filterProps` and `rootEl`. |
+| `includeStyleReset` | boolean       | Includes a minimal CSS style reset                                                                 |
+| `rootEl`            | String        | HTML tag name of the rendered DOM node. Helps ensure that only valid attributes reach the DOM.     |
+| `withProps`         | Object        | <key, value> pairs of props to apply to a component                                                |
```

Comment:
This table makes it much easier to identify/find what's possible in the options object. Nicely done!